src/pages/workspaces/workspace/analysis/modals/ComputeModal.js

```diff
@@ -19,7 +19,7 @@ import { withErrorReporting, withErrorReportingInModal } from 'src/libs/error'
 import Events, { extractWorkspaceDetails } from 'src/libs/events'
 import { betaVersionTag } from 'src/libs/logos'
 import * as Nav from 'src/libs/nav'
-import { useOnMount } from 'src/libs/react-utils'
+import { useOnMount, useUniqueId } from 'src/libs/react-utils'
```

Comment:
Could you switch from using the `react-utils` version of useUniqueId and instead import from IdContainer?

I the `useUniqueId` export from IdContainer will be what we use going forward.

Reply:
(as mentioned in another comment) this duality will be cleaned up in an outstanding UIE ticket:
https://broadworkbench.atlassian.net/browse/UIE-31